subscription-engine/kafka-ws/app/StateManager.js

```diff
@@ -0,0 +1,13 @@
+const { ServiceStateManager } = require('@dojot/microservice-sdk');
+const { getConfig, transformObjectKeys } = require('@dojot/microservice-sdk/lib/configManager');
+const { camelCase } = require('lodash');
+
+class StateManager {
+  constructor() {
+    const { lightship } = getConfig('KAFKA_WS');
+    const serviceConf = { lightship: transformObjectKeys(lightship, camelCase) };
+    this.serviceStateManager = new ServiceStateManager(serviceConf);
+  }
+}
+
+module.exports = new StateManager().serviceStateManager;
```

Comment:
If you really need this singleton, what about export it as a normal class and in the `index.js` you export the instance? Isn't it a better way to facilitate the unit tests? The same can be applied to the WebsocketTarball one.
You could create a directory inside `app` named `singletons` and then all the singletons you export in the `singletons/index.js` file.